### DISCUSSION TITLE
Getting Sony sw3 to be collector

### DISCUSSION THREAD

**Discussion by Npavkov** [NightscoutFoundation/xDrip#2906] [Q&A]
I currently have a setup of xdrip+ with a collector of libre2 )patched app), and I currently have it setup and working, and feeding the data to my sw3 watchface, however I would really like to get my sony sw3 watch to become collector when the phone is out of range. Is that able to be done in xdrip+ using libre2 patched app as hardware collector?

**Comment by psonnera** [comment]
Not sure this is possible without a bridge (miaomiao)
